packages/suite-desktop-api/tsconfig.lib.json

```diff
@@ -1,17 +1,10 @@
 {
-    "compileOnSave": false,
+    "extends": "./tsconfig.json",
     "compilerOptions": {
         "module": "commonjs",
         "target": "esnext",
         "outDir": "./lib",
-        "strict": true,
-        "declaration": true,
-        "removeComments": true,
-        "noUnusedLocals": true,
-        "noUnusedParameters": true,
-        "noErrorTruncation": true,
-        "types": ["jest", "node"]
+        "composite": false
     },
-    "include": ["./src/main.ts", "./src/renderer.ts"],
-    "exclude": ["node_modules"]
+    "include": ["./src"]
```

Comment:
exclude unit `__tests__` and `tsconfig.lib.tsbuildinfo`

Reply:
done, `.tsbuildinfo` is not produced at all for `lib` build now